File: PULSE.Model/ShippingInfo.cs
namespace PULSE.Model
{
    public class ShippingInfo : BaseModel
    {
        public string? Country { get; set; }
        public string? StateOrProvince { get; set; }
        public string? City { get; set; }
        public string? ZipCode { get; set; }
        public string? StreetAddress { get; set; }
    }
}

File: pulse_admin/lib/api/types/generic/shipping_info.dart
import 'package:json_annotation/json_annotation.dart';

part 'shipping_info.g.dart';

@JsonSerializable()
class ShippingInfo {
  String? country;
  String? stateOrProvince;
  String? city;
  String? streetAddress;
  String? zipCode;

  ShippingInfo();

  ShippingInfo.build(
      {this.country,
      this.stateOrProvince,
      this.city,
      this.streetAddress,
      this.zipCode});

  factory ShippingInfo.fromJson(Map<String, dynamic> json) =>
      _$ShippingInfoFromJson(json);

  Map<String, dynamic> toJson() => _$ShippingInfoToJson(this);
}
